### PR TITLE
WIP Comments on package page

### PR DESCRIPTION
**Pull request by nl0**
Add comments section to the package page.

Depends on:
  - ~#587~
  - ~#588~
  - ~#599~
  - ~#602~
  - #607 

### Questions

- [x] do we need the ability to edit and remove comments?
  - no
- [x] if yes, should package owner be able to do that, or author only?

### TODO

- [x] wire up redux stuff
- [x] wire up the api
- [x] intl
- [x] handle anonymous users
- [x] investigate XSS injection possibility
  - looks like we're safe
- [x] forbid images
- [x] port `unset` (reduxTools) from #518 
  - factored out in #599

### CODE REVIEW

**Comment by akarve** [comment]
A few notes:
* I think you are always using `<Markdown>` so should inherit XSS injection protection there? Would also be good to see if we can do anything to improve XSS protection
* We should forbid images in user-submitted markdown

**Comment by akarve** [comment]
Strings should be:
* "Basic markdown is supported"
* Left, non-default button should toggle between "Edit" and "Preview"

File: catalog/app/containers/Package/saga.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      method: 'POST',
+      body: JSON.stringify({ comment }),
+    });
+    */
+    // eslint-disable-next-line no-console
+    console.log('addComment', { owner, name, comment });
+    const result = {
+      author: 'admin',
+      date: new Date(),
+      isOwner: true,
```
[the remark this answers is not in the document]
Comment (dimaryaz) [reply catalog/app/containers/Package/saga.js]: Though if it makes things easier, I can make the server return it.

[the remark this answers is not in the document]
Comment (nl0) [reply catalog/app/containers/Package/saga.js]: @dimaryaz 
> Does isOwner mean, the comment author is the owner of the package the comment is attached to?

yep

> Either way, do we really need it? Just check if author == owner, etc.

you're right, i already have this data on the front end, so it's not required on the comment objects

**Comment by akarve** [comment]
Endpoint implemented in #607

**Comment by nl0** [comment]
@akarve rtm

**Comment by akarve** [comment]
v. minor request: the comment timestamp maybe should use relative time like we do under Lastest revision?

screen shot 2018-05-16 at 11 16 52 am

**Pull request merged**